docs/flank-output-investigation/flank_current_output.md

```diff
@@ -0,0 +1,310 @@
+      type: xctest
+      app: 
+      test-special-entitlements: false
+      fail-fast: false
+
+    flank:
+      max-test-shards: 1
+      shard-time: -1
+      num-test-runs: 1
+      smart-flank-gcs-path: 
+      smart-flank-disable-upload: false
+      default-test-time: 120.0
+      use-average-test-time-for-new-tests: false
+      test-targets-always-run:
+      files-to-download:
+      keep-file-path: false
+      full-junit-result: false
+      # iOS flank
+      test-targets:
+      disable-sharding: false
+      project: flank-open-source
+      local-result-dir: results
+      run-timeout: -1
+      ignore-failed-tests: false
+      output-style: single
+      disable-results-upload: false
+      default-class-test-time: 240.0
+      disable-usage-statistics: false
+      only-test-configuration: 
+      skip-test-configuration: 
+      output-report: none
+WARNING: Google cloud storage result directory should be unique, otherwise results from multiple test matrices will be overwritten or intermingled
+
+```
+
+### Run tests section
+
+```bash
+
+RunTests
+Found xctest: /Users/adamfilipowicz/Repos/flank/test_runner/src/test/kotlin/ftl/fixtures/tmp/ios/EarlGreyExample/Debug-iphoneos/EarlGreyExampleSwift.app/PlugIns/EarlGreyExampleSwiftTests.xctest
+isMacOS = true (mac os x)
+nm -U "/Users/adamfilipowicz/Repos/flank/test_runner/src/test/kotlin/ftl/fixtures/tmp/ios/EarlGreyExample/Debug-iphoneos/EarlGreyExampleSwift.app/PlugIns/EarlGreyExampleSwiftTests.xctest/EarlGreyExampleSwiftTests"
+nm -gU "/Users/adamfilipowicz/Repos/flank/test_runner/src/test/kotlin/ftl/fixtures/tmp/ios/EarlGreyExample/Debug-iphoneos/EarlGreyExampleSwift.app/PlugIns/EarlGreyExampleSwiftTests.xctest/EarlGreyExampleSwiftTests" | xargs -s 262144 xcrun swift-demangle
+
+ Smart Flank cache hit: 0% (0 / 17)
+  Shard times: 2040s
+
+  Saved 1 shards to /Users/adamfilipowicz/Repos/flank/results/test_dir/ios_shards.json
+  Uploading [ios_shards.json] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/test_dir/...
+
+  17 tests / 1 shard
+
+  Uploading [EarlGreyExample.zip] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/test_dir/.....
+  Uploading [EarlGreyExampleSwiftTests_shard_0.xctestrun] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/test_dir/shard_0/...
+  Uploading [session_id.txt] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/test_dir/...
+  1 matrix ids created in 0m 29s
+  Raw results will be stored in your GCS bucket at [https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/test_dir]
+
+```
+
+### Matrices webLink section
+
+```bash
+
+Matrices webLink
+  matrix-3d331uzs97mlt https://console.firebase.google.com/project/flank-open-source/testlab/histories/bh.a3b607c9bb6d0088/matrices/5523514684002242128/executions/bs.70a259d113387c0c
```

Comment:
I do not like indentations in console outputs. IMO it looks sloppy especially when comes to fold long lines on narrow screens. Take a look at Gclouds output, they have almost no indentations.
WDYF?